github/github.go

```diff
@@ -136,16 +139,17 @@ type Client struct {
 	Gists          *GistsService
 	Git            *GitService
 	Gitignores     *GitignoresService
+	Licenses       *LicensesService
```

Comment:
`Licenses` after `Issues`, please. :smile:

Reply:
Will it ever end... :P Done